spec/tests/require.fmf

```diff
@@ -22,6 +23,14 @@ description: |
     both the backward-compatible syntax ``library(repo/lib)``
     which fetches libraries from the `default location`__ as well
     as provide a ``dictionary`` with a full `fmf identifier`__.
+    For the latter case specify ``type: library``.
+
+    When referencing local files or directories use ``type: file``
+    and define list of paths relative to the fmf root directory.
+    These can be regular expressions to match multiple files or
+    directories or just a single file or directory name. By
+    default everything under test :ref:`/spec/tests/path` is
+    copied over to the system.
```

Comment:
Would that mean that content outside of the test directory wont be available? That would break our tests and given how much time Is needed to roll out new tmt version across our infra it could even mean that tests could be broken for days or weeks.

Reply:
sure, I'l add a comment that this will be actually pruned in a different patch in a following version of tmt

@kkaarreell for now it will copy everything, this patch allows you to add the definition of required files to your metadata before the pruning is actually done, I already have a patch that does it for libraries https://github.com/teemtee/tmt/pull/1723 but that requires a rebase on this patch and unlikely to be merged even in 1.24, for tests this will probably change as well together with libraries given the time it took already
but either way my guess is both of these are an item for 1.25 happening in 2 months-ish

Reply:
Good point, we need to make sure that changes required to keep copying files (== adding type: file + pattern) doesn't break scheduling with at that time deployed tmt  in FTF.